### PR TITLE
Feature/adopt typescript

### PR DESCRIPTION
**Pull request by mwoodffdc**
- Update Orizuru to use typescript

### CODE REVIEW

File: test/index/validator/shared/schema.test.ts [hunk 1]
```diff
@@ -22,32 +22,34 @@
...
-const
-	avsc = require('avsc'),
-	chai = require('chai'),
+import { SchemaValidator } from '../../../../src/index/validator/shared/schema';
 
-	expect = chai.expect,
+const expect = chai.expect;
 
-	schemaValidator = require('../../../../lib/index/validator/shared/schema');
+describe('index/validator/shared/schema.ts', () => {
```
Comment (dfruddffdc) [inline test/index/validator/shared/schema.test.ts]: OPTIONAL: Wouldn't bother putting .ts extensions - it's just noise when every test has this

File: test/index/validator/server.test.ts [hunk 2]
```diff
@@ -22,26 +22,23 @@
...
-describe('index/validator/server.js', () => {
+describe('index/validator/server.ts', () => {
 
 	describe('constructor', () => {
 
 		it('should return the schema if it is valid', () => {
 
 			// Given
-			const config = {
+			const options: any = {
```
Comment (dfruddffdc) [inline test/index/validator/server.test.ts]: COMMENT: Please avoid inflating what is already a big diff by renaming variables without legitimate reason (if there is a good reason, I digress).

File: test/index/server.test.ts [hunk 3]
```diff
@@ -22,79 +22,80 @@
...
+	});
 
 	afterEach(() => {
 		sinon.restore();
 	});
 
 	describe('constructor', () => {
 
-		it('should emit an error event if the configuration is invalid', () => {
+		it('should emit an error event if the optionsuration is invalid', () => {
```
Comment (dfruddffdc) [inline test/index/server.test.ts]: ACTION: Search and replace trouble.

Comment (mwoodffdc) [reply test/index/server.test.ts]: Sorted

File: test/index/validator/shared/schema.test.ts (hunk 1 above)
Comment (mwoodffdc) [reply test/index/validator/shared/schema.test.ts]: Sorted

**Pull request merged**